ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthFeeHistory.java

```diff
@@ -235,6 +254,46 @@ private List<Wei> calculateRewards(
     return rewards;
   }
 
+  /**
+   * This method returns a list of bounded rewards.
+   *
+   * @param rewards The list of rewards to be bounded.
+   * @return The list of bounded rewards.
+   */
+  private List<Wei> calculateBoundedRewards(final List<Wei> rewards) {
```

Comment:
will the default (no bounds) be used most of the time?

Reply:
yes